execution/executor/src/tests/mod.rs

```diff
@@ -213,7 +229,21 @@ fn test_executor_two_blocks_with_failed_txns() {
     let output2 = executor
         .execute_block((block2_id, block(block2_txns)), block1_id)
         .unwrap();
-    let ledger_info = gen_ledger_info(77, output2.root_hash(), block2_id, 1);
+
+    #[cfg(not(feature = "consensus-only-perf-test"))]
```

Comment:
Same here. I'd just ignore the entire test. This kind of compilation flagging in tests is probably not worth it.